commercetools-models/src/test/java/io/sphere/sdk/products/commands/ProductUpdateCommandIntegrationTest.java

```diff
@@ -1153,6 +1154,9 @@ public void productProjectionCanBeUsedToUpdateAProduct() throws Exception {
 
             final ProductProjection productProjection = client().executeBlocking(ProductProjectionByIdGet.of(product, STAGED));
 
+            SphereJsonUtils.toJsonString(productProjection);
+            SphereJsonUtils.toJsonNode(productProjection).forEach(System.out::println);
+
```

Comment:
Do we need the two lines above? Or are they left overs from testing?

Reply:
yep 😢 
i forgot to remove them.